Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/celestiaorg/celestia-app:v0.13.2 AS celestia-app
+FROM ghcr.io/celestiaorg/celestia-app:0.13.3 AS celestia-app
```

Comment:
@smuu can you follow up on this? did we intentionally drop the v?

Reply:
`v0.13.3` or `0.13.3` both are fine.